servers/rendering/shader_language.cpp

```diff
@@ -7595,7 +7607,7 @@ Error ShaderLanguage::_parse_shader(const Map<StringName, FunctionInfo> &p_funct
 	int instance_index = 0;
 #ifdef DEBUG_ENABLED
 	int uniform_buffer_size = 0;
-	int max_uniform_buffer_size = RenderingDevice::get_singleton()->limit_get(RenderingDevice::LIMIT_MAX_UNIFORM_BUFFER_SIZE);
+	int max_uniform_buffer_size = _get_device_limit(RenderingDevice::LIMIT_MAX_UNIFORM_BUFFER_SIZE);
```

Comment:
Instead of a dedicated method, maybe it would be better to just put it inline if there's only one use for it?
```suggestion
	int max_uniform_buffer_size = 0;
	if (RenderingDevice::get_singleton()) {
		max_uniform_buffer = RenderingDevice::get_singleton()->limit_get(RenderingDevice::LIMIT_MAX_UNIFORM_BUFFER_SIZE);
	}
```

Reply:
Hmm, I think a dedicated method is better (to prevent future code duplication), because this method may be needed in the future to count other constants besides uniform buffer size.

Reply:
Okay, changed it for now

Reply:
Note that if you thought it was better with a dedicated method, that was also fine. What I comment are just suggestions and as the maintainer of that code you're allowed to disagree with them :)

Reply:
I understand but this may be introduced later when the new usages of that function happen.